docs/release-announcement.md

```diff
@@ -0,0 +1,16 @@
+# Release announcement
+ 
+## Azure Industrial IoT Platform 2.7.200 
+ 
+The Azure Industrial IoT Platform is a Microsoft suite of modules and services that make it easier to connect your industrial assets to Azure. The Azure Industrial IoT Platform, consisting of edge and cloud services, allows you to discover industrial assets on-site and automatically register them in the cloud for easy access to your plant floor telemetry wherever you happen to be in the world.  We have worked to establish a large and vibrant partner network in order to support all types of industrial interfaces through the use of adapters, fully integrated with our platform – while natively supporting OPC UA by default. 
+ 
+We are pleased to announce that our next update, 2.7.200 of the Azure Industrial IoT Platform, will be released on January 12th. 
```

Comment:
1. As discussed, the release will most probably not have `2.7.200` exact version, as the version is based on the number of commits contained in it.
2. I would say that January 12th is a tentative date. We have no guarantee that it will be the exact date.

Reply:
Thanks Karapet, I changed it to 2.7.20x and updated the wording.